src/components/views/donate/RecurringDonationCard.tsx

```diff
@@ -224,10 +223,13 @@ export const RecurringDonationCard = () => {
 		}
 	}, [selectedToken, address, tokenStreams, project.anchorContracts]);
 
-	console.log(
```

Comment:
why was this log removed?

Reply:
@jainkrati, we removed it because it's now unnecessary. We initially added it to check our donation amount, but now we're confident everything is functioning correctly.